cf/query.py

```diff
@@ -905,7 +914,16 @@ def _evaluate(self, x, parent_attr):
             if _wi is not None:
                 return _wi(value)
 
-            return (x >= value[0]) & (x <= value[1])
+            open_lower, open_upper = self._open_bounds
+            if open_lower:
+                lower_bound = (x > value[0])
+            else:
+                lower_bound = (x >= value[0])
+            if open_upper:
+                upper_bound = (x < value[1])
+            else:
+                upper_bound = (x <= value[1])
+            return lower_bound & upper_bound
```

Comment:
Just some house style stuff:
```suggestion
            if open_lower:
                lower_bound = x > value[0]
            else:
                lower_bound = x >= value[0]

            if open_upper:
                upper_bound = x < value[1]
            else:
                upper_bound = x <= value[1]

            return lower_bound & upper_bound
```